src/nvme/nbft.h

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+/*
+ * This file is part of libnvme.
+ * Copyright (c) 2021-2022, Dell Inc. or its subsidiaries.  All Rights Reserved.
+ *
+ * Authors: Stuart Hayes <Stuart_Hayes@Dell.com>
+ *
+ */
+#ifndef _NBFT_H
+#define _NBFT_H
+
+#include <uuid/uuid.h>
+#include <ccan/list/list.h>
```

Comment:
`ccan` headers are not shipped as part of `libnvme` public headers nor pulled as a dependency externally. This will cause compilation errors when included from third party.